tests/conftest.py

```diff
@@ -248,33 +248,35 @@ def patched_get_exit_signal(pair, timeframe, dataframe, is_short):
     freqtrade.exchange.refresh_latest_ohlcv = lambda p: None
 
 
-def create_mock_trades(fee, is_short: bool = False, use_db: bool = True):
+def create_mock_trades(fee, is_short: Optional[bool] = False, use_db: bool = True):
```

Comment:
Do u want me to change this for all ? 
create_mock_trades(fee, is_short) -> create_mock_trades(fee, is_short=is_short) ?


I think we should make optional as the last arg.
def create_mock_trades(fee, is_short: Optional[bool] = False, use_db: bool = True): 
-> def create_mock_trades(fee, use_db: bool = True), is_short: Optional[bool] = False: 

I need to change all the test to -> create_mock_trades(fee, is_short=is_short) 

Otherwise it will break alot of tests

Reply:
Just make sure you're testing all the cases

Reply:
I think you're miss-reading what `Optional` means - it's not that the argument becomes Optional (must not be provided) - but it can become `None` (which `use_db` cannot).

Both `is_short` and `use_db` are optional parameters, as both have default values set - so you can easily use `create_mock_trades(fee)` without breaking anything.

Reply:
An `Optional` is a datatype and the type of it's value either has the type specified (int, bool, whatever) or it's value is None

Reply:
Thanks guys. Commit coming your way ;)